switchmap/server/db/ingest/update/device.py

```diff
@@ -435,12 +435,10 @@ def vlanport(self, test=False):
         # Insert rows
         if bool(inserts) is True:
             if bool(test) is False:
-                _vlanport.insert_row(inserts)
+                _vlanport.bulk_insert_rows(inserts)
```

Comment:
1. Revert.
2. Update `db.add_all` to use the new `bulk_insert_rows` method
3. Apply this to all the` switchmap/server/db/table` files. It's simpler. There v2.0 of sqlalchemy has a `bulk_insert_rows` replacement.